test/tests/trackers.js

```diff
@@ -197,4 +173,18 @@
       });
   });
 
+  QUnit.test("Whitelists rules", (assert) => {
+      bkg.settings.updateSetting('trackerBlockingEnabled', true);
+      
+      let testTab = Object.assign({}, JSON.parse(JSON.stringify(fakeTab)))
```

Comment:
why do we need `Object.assign` here? `JSON.parse(JSON.stringify` should be enough to create a deep copy of an object